tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "lib/*": ["../lib/*"],
```

Comment:
This needs to go up one level as the paths here are based on the baseUrl (this was just a random guess I made, but it turned out to fix the issue)